war/src/main/js/add-item.js

```diff
@@ -1,5 +1,4 @@
-// Initialize all modules by requiring them. Also makes sure they get bundled (see gulpfile.js).
-var $ = require('jquery-detached').getJQuery();
```

Comment:
Do plugins that depend on `jquery-detached` need updates as well?

Reply:
They don't. The code still works for plugins that use js-builder, js-modules and jquery-detached. With this update, Jenkins core just bundles its own dependencies on the vendors.js folder. Plugins using the older system still work as expected. 

FWIW, I have manually verified that the pipeline stageview and warnings-ng still work.

Reply:
Yes, I also checked that my plugin is still working 🎉.

Reply:
Closing this one then!